docs/source/conf.py

```diff
@@ -60,14 +59,20 @@
 napoleon_numpy_docstring = False
 napoleon_google_docstring = True
 
+
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 #
-# source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = {
+    '.rst': 'restructuredtext',
+    '.txt': 'markdown',
+    '.md': 'markdown',
+}
```

Comment:
Is this needed?

Reply:
It was by default supporting only `.rst`. I guess it is needed. I will have a look once.

Reply:
we don't have anything more than rst sources at the moment, do we?

Reply:
We could possibly move `Contributing.md` and `code of conduct.md` to docs, so that people can better know!
(They are moved for pytorch.org)

So we can possibly keep `md`.

Reply:
Again, let's please keep things minimal: we don't include .md files right now, so we don't need this.

When and *if* we include md files in the docs, we can revisit this decision.

TBF I'm not convinced about moving the contributing guide and the COC, as they're usually expected to be in the root of the GH repo. And all other domains do it this way too.